vpr/CMakeLists.txt

```diff
@@ -74,6 +74,8 @@ endif()
 
 set_target_properties(libvpr PROPERTIES PREFIX "") #Avoid extra 'lib' prefix
 
+target_include_directories(libvpr PUBLIC  ${PROJECT_SOURCE_DIR}/../libs/librrgraph/src/base/ ) 
```

Comment:
No need to modify this line. You just need to ensure that ``librrgraph`` is in the list of linked library.